llvm/include/llvm/Object/COFF.h

```diff
@@ -1362,6 +1362,47 @@ class SectionStrippedError
   SectionStrippedError() { setErrorCode(object_error::section_stripped); }
 };
 
+inline std::optional<std::string>
+getArm64ECMangledFunctionName(StringRef Name) {
```

Comment:
Would it be possible to re-use this in the existing code, instead of keeping the other copy?

Reply:
I added removing the copy in the new version (I will squash when pushing).